rerpc_test.go

```diff
@@ -423,6 +475,19 @@ func TestServerProtoGRPC(t *testing.T) {
 			assert.Zero(t, rerr.Details(), "error details")
 		})
 	}
+	testBadRoute := func(t *testing.T, client pingpb.PingServiceClientReRPC) {
+		t.Run("bad_route", func(t *testing.T) {
+			req := &pingpb.PingRequest{}
+			res, err := client.Ping(context.Background(), req, rerpc.OverrideProtobufPackage("test.badroute"))
```

Comment:
Awesome, I hadn't thought of using `OverrideProtobufPackage` this way but it's a good idea!